Source/Builders/Initialize-ISHImage.ps1

```diff
@@ -60,6 +60,7 @@ $mockOSUserPassword="Password123"
 $mockOSUserCredential=New-Object System.Management.Automation.PSCredential($mockOSUserName,(ConvertTo-SecureString $mockOSUserPassword -AsPlainText -Force))
 if($mockDatabase)
 {
+    & $dbScriptsPath\Install-MockDatabase.ps1 -ISHVersion $ishVersion
```

Comment:
Do not install as part of the initialize. it was a wrong approach at the beginning. Any special installation such as SQL should be explicitly referenced by the packer file which will provision the matching dedicated script.